diff --git a/node_modules/astro-pure/components/basic/Header.astro b/node_modules/astro-pure/components/basic/Header.astro
index 3101092..f72b9bc 100644
--- a/node_modules/astro-pure/components/basic/Header.astro
+++ b/node_modules/astro-pure/components/basic/Header.astro
@@ -10,7 +10,7 @@ import { Icon } from '../user'
   <a
     class='z-30 text-xl font-medium group-[.not-top]:ms-2 sm:group-[.not-top]:ms-3'
     style='transition:margin-inline 0.3s'
-    href='/'
+    href='/astroblog/'
     aria-label='Brand'>{config.title}</a
   >
 
@@ -34,7 +34,7 @@ import { Icon } from '../user'
           ))
         }
         <div class='flex w-full grow flex-row justify-end gap-x-3 sm:w-fit sm:gap-x-5'>
-          <a class='px-1 py-2 transition-none sm:px-2' href='/search' title='Search'>
+          <a class='px-1 py-2 transition-none sm:px-2' href='/astroblog/search' title='Search'>
             <span class='sr-only'>Search</span>
             <Icon name='search' class='size-5' />
           </a>
